test/utils/utils.go

```diff
@@ -54,6 +56,50 @@ func GeneratePldStatus(
 	return &clusterv1beta1.PlacementDecisionStatus{Decisions: plrDecision}
 }
 
+func RemovePolicyTemplateDBAnnotations(g Gomega, plc *unstructured.Unstructured) {
```

Comment:
(Optional) This should return the error rather than assert on it.
```suggestion
func RemovePolicyTemplateDBAnnotations(g Gomega, plc *unstructured.Unstructured) error {
```

Reply:
Fixed in the latest push!